cogs/music/basic_commands.py

```diff
@@ -439,3 +439,19 @@ async def _forceplay(self, ctx, *, query: str):
 
     if not player.is_playing:
         await player.play()
+
+
+@commands.command(name='forcedisconnect')
+@checks.dj_or(alone=True)
+async def _forcedisconnect(self, ctx):
+    """ Searches and plays a song from a given query. """
+    try:
+        player = self.bot.lavalink.player_manager.get(ctx.guild.id)
+        player.queue.clear()
+        await player.stop()
+    except Exception:
+        self.logger.exception("Error forcedisconnecting")
```

Comment:
Probably worth logging the actual exception too.

Reply:
Seems
https://www.loggly.com/blog/exceptional-logging-of-exceptions-in-python/